on-premise-docker-registry/README.md

```diff
@@ -0,0 +1,191 @@
+# Overview
+
+Many a time, customers would like to innovate by building applications and deploying them on Kyma runtime. Since Kyma is based on Kubernetes, they must package their application binaries as Docker images. These images then need to be stored on a Docker registry server.
+
+The customers have the option to use a third-party Docker registry hosted on public clouds such as:
+
+- Google Container Registry
+- Azure Container Registry
+- AWS Elastic Container Registry
+- and other vendor offerings
+
+Although this is a reliable option, many a time, customers do not wish to store their artifacts on public clouds. This could be due to various reasons, such as:
+
+- Security and compliance requirements that prohibit storing artifacts on public clouds
+- Customers wishing to use the bundled on-premise Docker registry licenses with their source versioning offerings.
+- Customers who do not wish to set up contract with public cloud vendor.
+- other reasons...
+
+These customers can consider leveraging the on-premise Docker registry. Docker images are pulled then using connectivity proxy and Cloud Connector.
+
+See the following sample flow of how this can be achieved
+
+![flow](assets/on-prem-docker-reg.png)
```

Comment:
There should be no need to store the png as well, you can show the svg directly, can't you?

Reply:
for icons coming from BTP, svg, does not display correctly.